filters/bevel/src/bevel.frag

```diff
@@ -0,0 +1,23 @@
+precision mediump float;
+
+varying vec2 vTextureCoord;
+uniform sampler2D uSampler;
+uniform vec4 filterArea;
+
+uniform float transformX;
+uniform float transformY;
+uniform vec3 lightColor;
+uniform float lightAlpha;
+uniform vec3 shadowColor;
+uniform float shadowAlpha;
+
+void main(void) {
+    vec2 transform = vec2(1.0 / filterArea) * vec2(transformX, transformY);
+    vec4 color = texture2D(uSampler, vTextureCoord);
+    float light = texture2D(uSampler, vTextureCoord - transform).a;
+    float shadow = texture2D(uSampler, vTextureCoord + transform).a;
```

Comment:
no filterClamp? prepare for problems. Padding is only partial solution.

Reply:
what kind of problems you mean?